app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/masterpasswordguidance/MasterPasswordGuidanceScreen.kt

```diff
@@ -0,0 +1,243 @@
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.material3.rememberTopAppBarState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.input.nestedscroll.nestedScroll
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.clearAndSetSemantics
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.x8bit.bitwarden.R
+import com.x8bit.bitwarden.ui.platform.base.util.EventsEffect
+import com.x8bit.bitwarden.ui.platform.components.appbar.BitwardenTopAppBar
+import com.x8bit.bitwarden.ui.platform.components.scaffold.BitwardenScaffold
+import com.x8bit.bitwarden.ui.platform.components.util.rememberVectorPainter
+import com.x8bit.bitwarden.ui.platform.theme.BitwardenTheme
+
+private const val BULLET_TWO_TAB = "\u2022\t\t"
+private const val TWO_THIRDS_WEIGHT = 0.66f
+
+/**
+ * The top level composable for the Master Password Guidance screen.
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Suppress("LongMethod")
+@Composable
+fun MasterPasswordGuidanceScreen(
+    onNavigateBack: () -> Unit,
+    onNavigateToGeneratePassword: () -> Unit,
+    viewModel: MasterPasswordGuidanceViewModel = hiltViewModel(),
+) {
+
+    EventsEffect(viewModel = viewModel) { event ->
+        when (event) {
+            MasterPasswordGuidanceEvent.NavigateBack -> onNavigateBack()
+            MasterPasswordGuidanceEvent.NavigateToPasswordGenerator -> {
+                onNavigateToGeneratePassword()
+            }
+        }
+    }
+
+    val scrollBehavior = TopAppBarDefaults.pinnedScrollBehavior(rememberTopAppBarState())
+    BitwardenScaffold(
+        modifier = Modifier
+            .fillMaxSize()
+            .nestedScroll(scrollBehavior.nestedScrollConnection),
+        topBar = {
+            BitwardenTopAppBar(
+                title = stringResource(id = R.string.master_password),
+                scrollBehavior = scrollBehavior,
+                navigationIcon = rememberVectorPainter(id = R.drawable.ic_close),
+                navigationIconContentDescription = stringResource(id = R.string.close),
+                onNavigationIconClick = remember(viewModel) {
+                    {
+                        viewModel.trySendAction(MasterPasswordGuidanceAction.CloseAction)
+                    }
+                },
+            )
+        },
+    ) { innerPadding ->
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .verticalScroll(rememberScrollState())
+                .padding(innerPadding)
+                .padding(horizontal = 16.dp),
+            verticalArrangement = Arrangement.spacedBy(16.dp),
```

Comment:
Using an `Arrangement` here has caused us issues when trying to manage custom spacing. we tend to stick with `Spacers`.

Reply:
This screen is so simple, I could take it or leave it here

Reply:
especially with that nav bar spacer, I think Arrangement is not the move, good call